oraclepaas/resource_mysql_service_instance.go

```diff
@@ -106,6 +100,19 @@ func resourceOraclePAASMySQLServiceInstance() *schema.Resource {
 				Sensitive: true,
 			},
 
+			"subnet": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"shape": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "oc3",
```

Comment:
is that a sensible default?

Reply:
I'd recomment removing the default, so the user must specify it explictly